generated/java/google-cloud-logging-v2/src/main/java/com/google/cloud/logging/spi/v2/ConfigClient.java

```diff
@@ -95,7 +94,6 @@
  * </pre>
  */
 @Generated("by GAPIC")
-@ExperimentalApi
```

Comment:
You're missing something here (toolkit change or googleapis change) - this was just recently added in.